json-serde/src/main/java/org/openx/data/jsonserde/JsonSerDe.java

```diff
@@ -80,11 +80,13 @@
     long serializedDataSize;
     // if set, will ignore malformed JSON in deserialization
     boolean ignoreMalformedJson = false;
+    boolean explicitNullValue = false;
 
     // properties used in configuration
     public static final String PROP_IGNORE_MALFORMED_JSON = "ignore.malformed.json";
     public static final String PROP_DOTS_IN_KEYS = "dots.in.keys";
     public static final String PROP_CASE_INSENSITIVE ="case.insensitive" ;
+    public static final String PROP_EXPLICIT_NULL ="explicit.null.value" ;
```

Comment:
call this explicit.null and the field explicitNull (lacking a better name)

Reply:
serialize.null.value may be better name?